.github/workflows/release.yml

```diff
@@ -20,20 +20,13 @@ jobs:
         uses: android-actions/setup-android@v2
       - name: Setup Android NDK
         run: yes | sdkmanager "ndk;21.0.6113669" >/dev/null
-      - name: Set default rust toolchain
-        run: rustup default 1.61.0
-      - name: Install rustfmt
-        run: rustup component add rustfmt
-      - name: Install clippy
-        run: rustup component add clippy
-      - name: Add armv7 target
-        run: rustup target add armv7-linux-androideabi
-      - name: Add aarch64 target
-        run: rustup target add aarch64-linux-android
-      - name: Add i686 target
-        run: rustup target add i686-linux-android
-      - name: Add x86_64 target
-        run: rustup target add x86_64-linux-android
+
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          components: clippy, rustfmt
+          # Add armv7, aarch64, i686, x86_64 targets
+          targets: armv7-linux-androideabi, aarch64-linux-android, i686-linux-android, x86_64-linux-android
```

Comment:
Should be tested on the next release.
- [ ] We should also consider running `cargo check` with the given targets to make sure we don't find issues when we want to release a new version